src/content/en/fundamentals/design-and-ui/ux-basics/index.md

```diff
@@ -0,0 +1,234 @@
+  <img src="images/project-map.jpg" alt="Project maps plot out the main steps for each user or player in a flow">
+  <figcaption>Project maps plot out the main steps for each user or player in a flow.</figcaption>
+</figure>
+
+This will give you a project map with the main steps in the process and is a great way to see the overview of the project without getting into specific details. It also allows others in the team to review the project map and see if it still matches the challenge statement. Later on when you will break down each step there will be more details but for now a project map will give you a higher level breakdown of the steps a user will take to complete their end goal.
+
+## Wireframing and storyboarding
+
+There is a method called crazy 8’s that involves folding a piece of paper twice over so that you have 8 panels. Then in each panel you draw out an idea based on all that you have learnt so far.
+
+When coming up with these sketch ideas you need to set yourself a short amount of time, say 10 minutes for 8 sketches. If you set yourself more than 20 minutes what you find is you will start to procrastinate, go make yourself a coffee, check email, have a general chat with your team and essentially avoid doing the work. We want to set a sense of urgency in this as if forces us to work quickly and more effectively.
+
+### Crazy 8’s
+
+The goal here is to give yourself 10 minutes to come up with lots of ideas, each idea filling a single panel. If you are working with a team get everyone to do their own as well. This process will kick start your brain and get you thinking about the challenge. Generally the sketch will be an interface design wireframe.
+
+From here present your ideas to your team and get them to present their ideas as well. Everyone must explain each idea that they came up with in detail and why they chose to go down a specific path. Try to reference back to the learnings as your justification as to why each idea could work. Once everyone has presented now comes the voting on ideas, each person gets two sticky dots and can vote on any idea. They can give both votes to a single idea if they really like it.
+
+<!-- <figure>
+  <img src="images/voting-ideas.jpg"   alt="You can take notes on post-its and vote on concepts or the sketches themselves">
+  <figcaption>You can take notes on post-its and vote on concepts or the sketches themselves.</figcaption>
+</figure> -->
+
+
+<figure  class="attempt-left">
+  <img src="images/crazy-8s.jpg" alt="Crazy 8s are a great way to get all of your ideas out onto page">
+  <figcaption>Crazy 8s are a great way to get all of your ideas out onto page.</figcaption>
+</figure>
+
+<figure class="attempt-right">
+  <img src="images/detailed-wireframe.jpg"   alt="Now you need to do a detailed design based on what you have learned">
+  <figcaption>Now you need to do a detailed design based on what you have learned.</figcaption>
+</figure>
+
+<div class="clearfix" style="height:0;"></div>
```

Comment:
Don't set a height of zero on this, just use the clearfix class. It's not rendering the clear fix and the next heading is getting lost here.